Gemfile.lock

```diff
@@ -178,7 +181,7 @@ GEM
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
     rainbow (2.2.1)
-    rake (12.0.0)
+    rake (11.3.0)
```

Comment:
Had to update rake to avoid an error:

```
NameError: uninitialized constant Gem::LOADED_SPECS_MUTEX
  /Users/starsirius/.rvm/gems/ruby-2.4.0@global/gems/bundler-1.14.6/lib/bundler/rubygems_integration.rb:426:in `block in replace_bin_path'
  /Users/starsirius/.rvm/rubies/ruby-2.4.0/bin/rake:22:in `<top (required)>'
```